events/ecr_scan.go

```diff
@@ -0,0 +1,30 @@
+package events
+
+type EcrScanEvent struct {
+	Version    string     `json:"version"`
+	ID         string     `json:"id"`
+	DetailType string     `json:"detail-type"`
+	Source     string     `json:"source"`
+	Time       string     `json:"time"`
+	Region     string     `json:"region"`
+	Resources  []string   `json:"resources"`
+	Account    string     `json:"account"`
+	Detail     DetailType `json:"detail"`
+}
+
+type DetailType struct {
```

Comment:
need to namespace sub-types, since the package events package is so broad. So this should be `ECRScanEventDetailType`